app/models/versioned/image/metadata_revision.rb

```diff
@@ -0,0 +1,42 @@
+# frozen_string_literal: true
+
+module Versioned
+  # A Image::MetadataRevision stores the data for an Image::Revision that when
```

Comment:
Don't think we need to repeat the class name in the comment :-). Can the comment focus on what it does do, rather than what it doesn't?

Reply:
Done.